Assets/Scripts/Internal/DaggerfallBillboardBatch.cs

```diff
@@ -967,6 +1044,10 @@ private void UpdateName()
 
         private bool ReadyCheck()
         {
+            // Get references
+            if (meshRenderer == null)
+                meshRenderer = GetComponent<MeshRenderer>();
+
```

Comment:
This is totally opportunistic change, hope you don't mind. Logic behind it: `GetComponent` is more costly that null check and `ReadyCheck()` is called repeatedly.

Reply:
That's no problems, cheers. :)